docs/getting-started.md

```diff
@@ -37,74 +28,41 @@ This recipe includes a simple illustration of particle composition: the Greet pa
-Greet(in Message message)
-```
-
-The demo only uses singleton views. Arcs also supports set views, e.g., `ShowProductList(in [Product] products)`. See [view.js](https://github.com/PolymerLabs/arcs/blob/master/runtime/view.js) for a complete description of the view interface.
+The recipe declares the particles that constitute (and run as part of) the program as well as how data flows through the particles in the Arc. Recipes can be hosted anywhere on the web and will eventually get crawled and indexed by Arcs. The same [manifest format](https://github.com/PolymerLabs/arcs/blob/master/runtime/manifest.md) is used to describe entity schemas (data types), views (sets of data), particles (code) and recipes (data flow).
 
+For this simple demo we have five different manifest files. One for every particle, one that defines the `Message` schema and one that pulls it all together and describes the recipe and the Arc. Take a look at the [arcs.manifest](https://glitch.com/edit/#!/arcs-hello-world?path=arc.manifest) file and its comments for a closer look at how the higher-level recipe is specified.
 
 ## Slots
 
-Slots are named UI placeholders where particles can render content into. E.g., the demo script exposes a root slot that the Greet particle renders into.
+Slots are named UI placeholders where particles can render content into. E.g., the Arcs playground exposes a root slot that the Greet particle renders into.
 
 Slots are also the mechanism by which UI composes in Arcs. E.g., the Greet particle renders DOM inside the root slot and exposes a new slot for another particle to render additional content into its DOM. The Greet DOM output contains the following tag: `<div slotid="customgreeting"></div>`. The PersonalGreet particle can then specify in its manifest that it wants to be rendered into the 'customgreeting' slot.
 
 TODO: describe more complex slot composition (e.g., set slots).
 
-
 ## Particles
 
-A particle is the basic unit of computation in Arcs. Simply put it's a JavaScript file that contains a class that inherits either from [DomParticle](https://github.com/PolymerLabs/arcs/blob/master/runtime/dom-particle.js) (if the particle renders DOM UI) or from its parent class [Particle](https://github.com/PolymerLabs/arcs/blob/master/runtime/particle.js) (if it only operates on input and output views).
-
-
-TODO: show the basic new format particle manifest and link to the spec.
+A particle is the basic unit of computation in Arcs. Simply put it's a JavaScript file that contains a class that inherits either from [DomParticle](https://github.com/PolymerLabs/arcs/blob/master/runtime/dom-particle.js) (if the particle renders DOM UI) or from its parent class [Particle](https://github.com/PolymerLabs/arcs/blob/master/runtime/particle.js) (if it only operates on input and output views). Take a look at the [Greet](https://glitch.com/edit/#!/arcs-hello-world?path=particles/Greet/Greet.manifest) particle manifest and its accompanied JavaScript file.
```

Comment:
Maybe a direct link to the accompanied JS file (https://glitch.com/edit/#!/arcs-hello-world) at the end of the last sentence?

Reply:
Added a link as suggested and moved the comment about serialized Arc a bit further down.